packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -73,6 +73,16 @@ abstract class VideoPlayerPlatform extends PlatformInterface {
     throw UnimplementedError('pause() has not been implemented.');
   }
 
+  /// The video enters fullScreen (only available for web platform).
```

Comment:
Functionality that is only ever expected to be relevant for a single platform should be added directly to the platform in question, not the platform interface.

It's not clear to me how this is intended to be used though, as `textureId` is a hidden property of the controller in the app-facing package. What's the user-facing design for this feature?

Reply:
Hi Stuart, I've added the rest of the code, fullscreen implemented in the facing app and platform tests. Please review, and let me know about the proceeding steps.